components/lang/src/main/java/io/cellery/impl/CreateImage.java

```diff
@@ -427,4 +446,15 @@ private void generateCellReference() {
             throw new BallerinaException("Error occurred while generating reference file " + targetFileNameWithPath);
         }
     }
+
+    private void createDockerImage(String imageName, String dockerDir) {
+        DockerModel dockerModel = new DockerModel();
+        dockerModel.setName(imageName);
+        try {
+            DockerArtifactHandler dockerArtifactHandler = new DockerArtifactHandler(dockerModel);
+            dockerArtifactHandler.buildImage(dockerModel, dockerDir);
+        } catch (DockerGenException | InterruptedException | IOException e) {
+            throw new BallerinaException("Error occurred while building Docker image " + e.getMessage());
```

Comment:
should we throw this the exception object itself without using e.getMessage()? Or at least log it, if the ballerina native API does not permit returning an exception object.